src/frontend/Navigation/Stack/AppScreens.tsx

```diff
@@ -353,5 +358,28 @@ export const createDefaultScreenGroup = ({
         options={{headerTitle: 'Create Test Data'}}
       />
     )}
+    <RootStack.Screen
+      name="RemoteArchiveOff"
+      component={RemoteArchiveOff}
+      options={{headerTitle: intl(RemoteArchiveOff.navTitle)}}
+    />
+    <RootStack.Screen
+      name="AddRemoteArchive"
+      component={AddRemoteArchive}
+      options={{
+        headerTitle: intl(AddRemoteArchive.navTitle),
+        headerRight: () => <SaveButton onPress={() => {}} isLoading={false} />,
```

Comment:
its a [weird quirk of react navigation](https://reactnavigation.org/docs/header-buttons/#header-interaction-with-its-screen-component). The header gets rendered first, but the body is what passes the onSubmit function to the header. So you need to render an empty button, and then in the useEffect of the body, you pass the onsubmit to the header. If you have no empty button, it will cause it to flicker.